Resources/public/js/views/universaldiscovery/ez-universaldiscoveryfinderexplorerview.js

```diff
@@ -214,6 +220,17 @@ YUI.add('ez-universaldiscoveryfinderexplorerview', function (Y) {
             },
 
             /**
+             * The discover root depth if the UDW is configured with one.
+             *
+             * @attribute discoverRootDepth
+             * @type {Number|false|Null}
```

Comment:
that's a tad too much :) `{Number|false}` (with default to `false`) seems enough to me.